### PR TITLE
Sidebar component fix with new login page

### PR DESCRIPTION
**Pull request by andreo141**
Hi guys. Can you test the responsiveness this time? On my pc it seems to work fine and i got rid of the previous bugs. Please let me know if you guys encounter any bugs.

### CODE REVIEW

**Review by DennisVanEecke — APPROVED** [review]
Works fine on my machine. Great improvement.

File: app/templates/core-data.hbs [hunk 1]
```diff
@@ -1,3 +1 @@
-<WithSidebar>
-  {{outlet}}
-</WithSidebar>
+<SidebarContainer />
```
Comment (lagartoverde) [inline app/templates/core-data.hbs:1]: The component should not include the outlet, the outlet should be passed down to the component, so we can include more things

**Review by lagartoverde — CHANGES_REQUESTED** [review]
Also a minor styling thing

File: app/templates/core-data.hbs (hunk 1 above)
Comment (andreo141) [reply app/templates/core-data.hbs]: I restored the components and templates to their previous structure (how you implemented the sidebar). The only thing i changed now is a added some media queries and styling. Let me know if there is still room for improvement.

**Review by YJouhar47 — APPROVED** [review]

**Review by lagartoverde — APPROVED** [review]
Good job

**Pull request merged**